icongenie/lib/utils/validate-profile-object.js

```diff
@@ -67,18 +53,30 @@ const assetsSchema = Joi.array().items({
   tag: Joi.string()
 })
 
-module.exports = function validateProfileObject (profileObject, generatingProfileFile = false) {
-  // ? - When generating the profile file, we don't want to validate with # on the hex color.
-  // : - When generating the icon, we're expecting a hash on the color (automatically added to user input via the CLI)
-  const colorParams = generatingProfileFile
-    ? colorParamsSchemaProfileSave
-    : colorParamsSchema
+/**
+ * When generating the profile file, we don't want to validate with # on the hex color.
+ * When generating the icon, we're expecting a hash on the color (automatically added to user input via the CLI)
+ */
+const getColorParamsSchema = (requireHash) => {
+  const colorPattern = Joi.string().pattern(new RegExp(`^${requireHash ? '#' : ''}[0-9A-Fa-f]{3}([0-9A-Fa-f]{3})?$`))
+  return {
+    themeColor: colorPattern,
+    pngColor: colorPattern,
+    splashscreenColor: colorPattern,
+    svgColor: colorPattern,
```

Comment:
_style:_
I guess we don't use dangling commas 🙃

Reply:
I guess it has been slipped through when copy/pasting from the middle of the old pattern object.

Reply:
Ah whoops. Will sort. Nice spot.